app/views/meeting.html

```diff
@@ -93,10 +93,10 @@ <h4 >{{ctrl.header}}</h4>
     </div>
 
     <div ng-if="mobile" class="ad-page-content">
-        <button type="button" class="btn btn-primary ad-header-button" ng-click="ctrl.topicClicked()" translate translate-cloak>STR_TOPIC</button>
-        <button type="button" class="btn btn-primary ad-header-button ng-class:[{'disabled':!ctrl.attachmentData.objects.length}]" ng-click="ctrl.attachmentClicked()"><span translate translate-cloak>STR_ATTACHMENTS</span> <span class="badge">{{ctrl.attachmentData.objects.length}}</span></button>
-        <button type="button" class="btn btn-primary ad-header-button ng-class:[{'disabled':!ctrl.additionalMaterialData.objects.length}]" ng-click="ctrl.additionalMaterialClicked()"><span translate translate-cloak>STR_ADDITIONAL_MATERIAL</span> <span class="badge">{{ctrl.additionalMaterialData.objects.length}}</span></button>
-        <button type="button" class="btn btn-primary ad-header-button ng-class:[{'disabled':!ctrl.decisionData.objects.length}]" ng-click="ctrl.decisionClicked()"><span translate translate-cloak>STR_DECISION_HISTORY</span> <span class="badge">{{ctrl.decisionData.objects.length}}</span></button>
+        <button type="button" ng-disabled="!ctrl.topicData" class="btn btn-primary ad-header-button" ng-click="ctrl.topicClicked()" translate translate-cloak>STR_TOPIC</button>
+        <button type="button" ng-disabled="!ctrl.attachmentData.objects.length" class="btn btn-primary ad-header-button" ng-click="ctrl.attachmentClicked()"><span translate translate-cloak>STR_ATTACHMENTS</span> <span class="badge">{{ctrl.attachmentData.objects.length}}</span></button>
```

Comment:
Notice following ctrl.XXXdata.objects.length will throw exceptions if the `ctrl.XXXdata` are null, or if their `.objects` property is null. Have you considered this?